### PR TITLE
[workspace] Remove privileged workspace config flag

### PR DESCRIPTION
**Pull request by csweichel**
This PR removes the `privileged` workspace config flag.

### CODE REVIEW

**Review by meysholdt — APPROVED** [review]
LGTM

**Comment by akosyakov** [comment]
We need to test that opening a workspace with a config containing such flag still works, but job is failing for me right now.

File: components/gitpod-protocol/data/gitpod-schema.json [hunk 1]
```diff
@@ -159,11 +159,6 @@
             "type": "string",
             "description": "Path to where the IDE's workspace should be opened."
         },
-        "privileged": {
```
Comment (akosyakov) [inline components/gitpod-protocol/data/gitpod-schema.json:162]: it should be also removed from public scheme: components/dashboard/public/schemas/gitpod-schema.json

**Review by akosyakov — APPROVED** [review]
I was able to start repos with this flag and without, other configurations were still used

**Pull request merged**